corn/index.html

```diff
@@ -6,11 +6,67 @@
   <meta charset="utf-8">
   <title>Todo App</title>
   <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&display=swap" rel="stylesheet">
   <!-- If you uncomment the following line the design MUST be dark -->
-  <!-- link rel="stylesheet" href="styles.black.css" / -->
+  <!--link rel="stylesheet" href="styles.black.css" /-->
 </head>
 
 <body>
-
+  <header>
+    <h1>
+      <time datetime="2020-03-09">March 9, 2020</time>
+      <p class="subtitle">5 incomplete, 5 completed</p>
+    </h1>
+  </header>
+  <hr />
+  <main>
+    <section>
+      <h2>Incomplete</h2>
+      <ul>
+        <li class="li">
+          <input type="checkbox"> Upload 1099-R to TurboTax
+          <p>💰 Finance</p>
+        </li>
+        <li class="li">
+          <input type="checkbox">Submit 2019 tax return
+          <p>💰 Finance</p>
+        </li>
+        <li class="li">
+          <input type="checkbox">Print parking passes
+          <p>💞 Wedding</p>
+        </li>
+        <li class="li">
+          <input type="checkbox">Sign contract, send back
+          <p>🖥️ Freelance</p>
+        </li>
+        <li class="li">
+          <input type="checkbox">Hand sanitizer
+          <p>🛒 Shopping List</p>
+        </li>
+      </ul>
+    </section>
+    <section>
+      <h2>Completed</h2>
+      <ul>
+        <label>
```

Comment:
This is wrong: child element of `<ul>` is `<li>`, not `<label>`. Also it make sense to use `<label>` for input elements.